packages/invoices/src/InboundShipment/DetailView/Footer.tsx

```diff
@@ -15,18 +15,15 @@ import {
   getStatusTranslator,
   inboundStatuses,
 } from '../../utils';
-import { InboundShipment } from '../../types';
-import {
-  useInboundFields,
-  useIsInboundEditable,
-} from 'packages/invoices/src/InboundShipment/DetailView/api';
+import { Invoice } from '../../types';
+import { useInboundFields, useIsInboundEditable } from './api';
 
 interface InboundDetailFooterProps {
-  draft: InboundShipment;
+  draft: Invoice;
```

Comment:
The `InboundShipment` object itself mainly was different because of the added methods attached to the object which are now unused